accord-core/src/main/java/accord/local/RedundantStatus.java

```diff
@@ -112,6 +114,11 @@ public enum Property
         // not persisted
         WAS_OWNED                          (false,  false, LT, LOCALLY_DEFUNCT),
         NOT_OWNED                          (false, false, LT),
+
+        /**
+         * TODO
+         */
+        UNSAFE_BEFORE                      (false, false, LT),
```

Comment:
LOCALLY_CORRUPT or LOCALLY_LOST or ...? Should imply PRE_BOOTSTRAP_OR_STALE, since we must be stale (we cannot execute anything for this range until we bootstrap)